classes/services/QueryBuilders/PKPSubmissionListQueryBuilder.php

```diff
@@ -0,0 +1,231 @@
+<?php
+
+/**
+ * @file classes/services/QueryBuilders/PKPSubmissionListQueryBuilder.php
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University
+ * Copyright (c) 2000-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class SubmissionListQueryBuilder
+ * @ingroup query_builders
+ *
+ * @brief Submission list Query builder
+ */
+
+namespace App\Services\QueryBuilders;
+
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+abstract class PKPSubmissionListQueryBuilder extends BaseQueryBuilder {
+
+	/** @var int Context ID */
+	protected $contextId = null;
+
+	/** @var array list of columns for query */
+	protected $columns = array();
+
+	/** @var string order by column */
+	protected $orderColumn = 's.date_submitted';
+
+	/** @var string order by direction */
+	protected $orderDirection = 'DESC';
+
+	/** @var array list of statuses */
+	protected $statuses = null;
+
+	/** @var int user ID */
+	protected $assigneeId = null;
+
+	/** @var string search phrase */
+	protected $searchPhrase = null;
+
+	/** @var bool whether to return only a count of results */
+	protected $countOnly = null;
+
+	/**
+	 * Constructor
+	 *
+	 * @param int $contextId context ID
+	 */
+	public function __construct($contextId) {
+		parent::__construct();
+		$this->contextId = $contextId;
+	}
+
+	/**
+	 * Set result order column and direction
+	 *
+	 * @param string $column
+	 * @param string $direction
+	 *
+	 * @return \App\Services\QueryBuilders\SubmissionListQueryBuilder
+	 */
+	public function orderBy($column, $direction = 'DESC') {
+		if ($column === 'lastModified') {
```

Comment:
illuminate/database takes care of that (cf. https://github.com/illuminate/database/blob/4.1/Query/Builder.php#L987)